File: src/main/java/com/appinbox/sdk/auth/Device.java
package com.appinbox.sdk.auth;

import com.appinbox.sdk.AppInboxSDK;

public class Device {
    private String id;
    private String sdkVersion;

    public Device(String id) {
        this.id = id;
        this.sdkVersion = AppInboxSDK.SDK_VERSION;
    }
}

File: src/main/java/com/appinbox/sdk/model/Device.kt
package com.appinbox.sdk.model;

import com.appinbox.sdk.AppInboxSDK;

data class Device (var id: String = "", val sdkVersion: String = AppInboxSDK.SDK_VERSION) {
    constructor(deviceId: String) : this() {
        id = deviceId
    }
}